tests/test_settings.cpp

```diff
@@ -745,6 +745,221 @@ TEST_CASE("Settings Save: LayerServer/None", "[settings]") {
     CHECK(srcSettings == cmpSettings);
 }
 
+TEST_CASE("Settings Load: LayerServer/All", "[settings]") {
```

Comment:
Not sure where this file comes from but it probably is not part of the PR

Reply:
Fixed in: [dd833a](https://github.com/OpenSpace/OpenSpace/commit/dd833aa38af348842bee61b73159e1ce3d393ad7)